File: csharp/BankAccount/BankAccount/Branch.cs
namespace Banking;

public class Branch
{
    public string Name { get; }

    public string BranchManager { get; }

    public PersonalAccountsManager PersonalAccounts { get; } = new PersonalAccountsManager();

    public Branch(string name)
    {
        Name = name;
        BranchManager = "Mr Gringotts Goblin";
    }

    public bool Withdraw(string customerName, decimal cashAmount)
    {
        return PersonalAccounts.Withdraw(customerName, cashAmount);
    }
}

File: java/src/main/java/banking/Branch.java
package banking;

class Branch {
    private final String name;
    private final String branchManager;
    private final PersonalAccountsManager personalAccounts = new PersonalAccountsManager();

    public Branch(String name) {
        this.name = name;
        this.branchManager = "Mr Gringotts Goblin";
    }

    public String getName() {
        return name;
    }

    public String getBranchManager() {
        return branchManager;
    }

    public PersonalAccountsManager getPersonalAccounts() {
        return personalAccounts;
    }
}
